Makefile

```diff
@@ -87,6 +87,32 @@ ifeq ($(VENV_OS), arm64mac)
 	echo "    finished bin-conda" >> $(LOGFILE)
 endif
 
+#>>>
+# make build-all -P
+
+#<<<
+.PHONY: build-all
+build-all:
+	./pre-build-check.sh
+
+# Setup the entire stack
+	$(MAKE) init-docker
+	$(MAKE) build-images
+	$(MAKE) compose
+	$(MAKE) init-authx
+
+
+#>>>
+	# (re)build service image for all modules
+	# add BUILD_OPTS='--no-cache' to ignore cached builds
+	# BUILD_OPTS='--no-cache' make build-$module
+	# make images
+
+	#<<<
+	.PHONY: build-images
+	build-images: #toil-docker
+		$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) build-$(MODULE);)
+
```

Comment:
I think an extra tab got added here which makes the `make build-images` step fail.

```suggestion
# (re)build service image for all modules
# add BUILD_OPTS='--no-cache' to ignore cached builds
# BUILD_OPTS='--no-cache' make build-$module
# make images

#<<<
.PHONY: build-images
build-images: #toil-docker
	$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) build-$(MODULE);)

```